skrifa/src/outline/autohint/style.rs

```diff
@@ -198,6 +237,16 @@ impl GlyphStyleMap {
                 }
             })
     }
+
+    fn use_style(&mut self, style_ix: usize) {
+        let mapped = &mut self.metrics_map[style_ix];
+        if *mapped == 0xFF {
```

Comment:
This is a bit cryptic with magic values...

Reply:
Oops, I meant to name this constant when I made the change.